src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/FailedDocUploadProcessorTest.java

```diff
@@ -71,15 +76,15 @@ public void should_successfully_reupload_documents_when_initial_blob_processor_w
         blobProcessorTask.processBlobs();
 
         // when
-        processor.processJurisdiction("SSCS");
+        processor.processJurisdiction("SSCS", serviceBusHelper);
```

Comment:
could `serviceBusHelper` simply be passed to the `processor` in constructor?

Reply:
It could, if it's worth it :-)

Reply:
Yes, it was worth it. Done.